api/javatests/org/openyolo/api/CredentialClientTest.java

```diff
@@ -76,6 +77,7 @@ public void testGetApplicationBoundInstance() throws Exception {
         assertNotNull(test);
     }
 
+    @SuppressWarnings("WrongConstant")
```

Comment:
Android studio complains that the call to queryIntentActivities should have a second parameter from a constant set. Since it's a mock call (passing `anyInt()`), it doesn't really matter.